rollbar-android/src/main/java/com/rollbar/android/Rollbar.java

```diff
@@ -346,6 +372,15 @@ public Config provide(ConfigBuilder builder) {
     });
   }
 
+  /**
+   * Handle all uncaught errors on all threads with the current notifier.
+   */
+  public void handleUncaughtErrors() {
```

Comment:
I'm concerned about the change in behavior without a major version bump. For Android, it hadn't worked at all, so I'm not worried about it. For other targets, going from per thread to global will be a big change for some setups.

Reply:
What's the change in behavior a part of having an uncaught handler for every thread instead of only for the current one in which the notifier is created?  I might be wrong, but looks like a fix as now we don't capture all uncaught exceptions...

Reply:
If you say change it, I'll change it. But yes, the existing interface both takes a thread object and specifies in the comment that it's per thread, so I took that to be fully intentional. https://github.com/rollbar/rollbar-java/blob/master/rollbar-java/src/main/java/com/rollbar/notifier/Rollbar.java#L80-L98

Reply:
Ok, I think we can be conservative with that change and don't do it.

Reply:
:+1: Sounds good. I like that this also keeps the current PR in the scope of the Android issue. 

If we separate the Android SDK, that might be a good time to plan a major version and update this.